responsive-grid-layout/dilara/global.css

```diff
@@ -32,4 +41,10 @@ body {
 .container {
   width: 100vw;
   height: 100vh;
+  display: grid;
```

Comment:
I know that I placed classes initially in this order by my intention was not to use the classes above.
When doing your own projects, try to always place the code for container element above children elements. This would make your code more readable and help to avoid unexpected behaviour  related to specificity of elements (see more here: https://www.w3schools.com/css/css_specificity.asp).